README.md

```diff
@@ -196,6 +196,10 @@ settings. If `sky` is missing, the default sky settings are used, if
 defaults to `6`.
 + `color`: optional, three RGB values in `[0,1]` range, in linear
 color space, defaults to `[0.855,0.863,1]`.
++ `ambientLight`: optional, an object that configures ambient
+lighting parameters. If `ambientLight` is missing, default ambient lighting
```

Comment:
lighting > light for consistency with the UI?

Reply:
Yes, Light will be better